lambda.repo.md

```diff
@@ -0,0 +1,46 @@
+##Lambda Repo Structure
+
+####The following describes the directory structure for the `lambda` repo:
+
+```
+lambda
+   |
+   |_ system
+         |
+         |_ eventA
+         |     |
+         |     |_ actionA
+         |     |     |
+         |     |     |_ lib
+         |     |     |_ support
+         |     |     |    |
+         |     |     |    |_ install
+         |     |     |    |_ samples
+         |     |     |         |
+         |     |     |         |_ sampleEventA
```

Comment:
Let's do `lambda/system/eventA/actionA/support/samples/events/sampleEventA.json` as the example